.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+on:
+  pull_request:
+    branches:
+      - production
+
+jobs:
+  lint-build-typecheck:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)